test/ibmq/test_account_client.py

```diff
@@ -163,6 +163,14 @@ def test_backend_properties(self, backend):
         properties = api.backend_properties(backend.name())
         self.assertIsNotNone(properties)
 
+    @requires_device
+    def test_backend_jobs_limit(self, backend):
+        """Check the backend job limits of a real backend."""
+        api = self._get_client()
+
+        jobs_limit = api.backend_jobs_limit(backend.name())
+        self.assertIsNotNone(jobs_limit)
```

Comment:
Can you also check that the attributes of `jobs_limit` are not `None` (and maybe not 0)?